usaspending_api/disaster/v2/views/award/amount.py

```diff
@@ -22,17 +27,24 @@ def queryset(self):
             self.all_closed_defc_submissions,
             self.has_award_of_provided_type,
             self.is_in_provided_def_codes,
-            self.is_non_zero_award_spending,
         ]
 
-        count_field = self.unique_file_c
         if self.award_type_codes:
             count_field = "award_id"
+        else:
+            count_field = self.unique_file_c
 
         fields = {
             "award_count": Count(count_field, distinct=True),
             "obligation": Coalesce(Sum("transaction_obligated_amount"), 0),
             "outlay": self.outlay_field_annotation,
         }
 
-        return FinancialAccountsByAwards.objects.filter(*filters).aggregate(**fields)
+        if self.award_type_codes:
```

Comment:
Seems like this could utilize the same pattern in `award/count/` instead of this conditional. Not a blocker to merge

Reply:
chalking up as tech debt